.github/workflows/ci.yml

```diff
@@ -110,7 +110,9 @@ jobs:
           rm -rf tokio/tests
 
       - name: miri
-        run: cargo miri test --features rt,rt-multi-thread,sync task
+        run: |
+          cargo miri test --features rt,rt-multi-thread,sync task
+          cargo miri test -p tokio-util 'sync::tests::mpsc'
```

Comment:
This is good, but we should have a miri test that calls `is_ready` and `clone` while the state is in either `Ready` or `Acquire`.

Reply:
Unfortunately it [seems](https://github.com/rust-lang/miri/issues/1038) that there is no way to poll futures under MIRI.
Of course, I plan to write some less trivial tests but there is no way to verify them in MIRI in the near future.